src/main/resources/weatherCacheConfig.xml

```diff
@@ -1,5 +1,8 @@
-<distributed-cache name="weather" remote-timeout="17500">
-    <locking acquire-timeout="15000" striping="false" concurrency-level="1000"/>
+<distributed-cache name="weather" statistics="true">
     <encoding media-type="application/x-protostream"/>
-    <state-transfer timeout="60000"/>
+    <indexing enabled="true" storage="filesystem" startup-mode="AUTO">
```

Comment:
nice to show an example of usage of a startup mode, in this case the auto mode is equivalent to purge, so an index purge at startup will run, since the cache is not persistent (no cache store) while the index uses local filesystem by default. (this is only clarification)